Adapter.java

```diff
@@ -0,0 +1,141 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Adapter {
+    private ApplictionContext applictionContext;
+
+    public Adapter(ApplictionContext applictionContext) {
+        this.applictionContext = applictionContext;
+    }
+
+    public void start() throws IOException {
+        System.out.println("Добро пожаловать в электронную библиотеку\n" + "чтобы перейти в раздел книги, нажмите 1\n"
+                + "чтобы перейти в раздел читатели, нажмите 2\n"+"чтобы перейти в раздел заказы, нажмите 3");
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+        String reader = bufferedReader.readLine();
+        int choice = Integer.parseInt(reader);
+        switch (choice) {
+            case 1:
+
+                System.out.println("чтобы выбрать книгу по ID, нажмите 1\n"
```

Comment:
Move all logic in every case into separate class which is responsible for that logic since your method body is too large